docs/introduction.md

```diff
@@ -4,145 +4,62 @@
-  capabilities which may be only suitable for specific user cases.
-  This makes extending such platform to leverage broader Kubernetes ecosystems
-  almost impossible.
-
-- It could be painful to develop new capability based on existing interfaces.
-  The operator design could easily becomes a marathon negotiation in order
-  to meet the abstraction requirement.
-
-In the end, application developers may complain that Kubernetes is hard to use
-and the platform builders may want to help but they cannot do it easily.
+The modern trend of cloud-native technology is moving towards pursuing consistent application delivery across clouds and on-premises infrastructures using Kubernetes as the common abstraction layer. Kubernetes, although excellent in abstracting away low-level infrastructure details, does introduce another layer of complexity to application developers, namely: pods, port exposing, privilege escalation, resource claims, CRD, and so on. We’ve seen such a learning curve and the lack of developer-facing abstraction have impacted user experience, slowed down productivity, led to unexpected errors or misconfigurations in production. People start questioning the value of this revolution: "why am I bothered with that many YAML files?" they asked.
+
+On the other hand, abstracting Kubernetes to serve developers' needs is a highly opinionated process, and the resultant abstractions would only make sense had the decision makers been the platform builders. Unfortunately, the platform builders today face the following dilemma:
+
+*There is no tool or framework for them to easily extend the abstractions if any*. 
+
+Thus, many platforms today introduce restricted abstractions, implementations and add-on mechanisms despite the extensibility of Kubernetes. This makes extending such platforms per developers' requirements or to wider scenarios almost impossible, not to mention taking full advantage of the extremely rich Kubernetes ecosystems. . 
+
+In the end, developers complain such a platform is too rigid and slows in response to feature requests or improvements. The platform engineers do want to help but the engineering cost is daunting: any simple API change in the platform could easily become a marathon negotiation around the opinionated abstraction design.
 
 ## What is KubeVela?
 
-KubeVela is a framework to help platform builders to easily expose and extend
-Kubernetes capabilities for application management. It is built on top of
-Kubernetes and relieves the pains of both the platform builders and the application
-developers by doing the following:
+For developers, KubeVela itself is an easy-to-use tool that enables them to describe and ship their applications to Kubernetes with minimal effort. Instead of managing a handful Kubernetes YAML files, a simple docker-compose style [Appfile](./docs/developers/devex/appfile.md) is what all they need, following an application-centric workflow that can be easily integrated with any CI/CD pipeline.
 
```

Comment:
- I think we should add something to link the above to the below。
`The above cannot be achievement without Kubevela's innovative offerings to platform builders. For platform builder, ...`